drf_braces/serializers/form_serializer.py

```diff
@@ -41,9 +41,7 @@ def run_validation(self, data):
             # Only handle a ValidationError if the full validation is
             # requested or if field is in minimum required in the case
             # of partial validation.
-            if any([not self.parent.partial,
-                    self.parent.Meta.failure_mode == FormSerializerFailure.fail,
-                    self.field_name in self.parent.Meta.minimum_required]):
+            if self.field_name in self.parent.Meta.minimum_required:
```

Comment:
how does this help? the original conditions are more strict when validating fields